packages/botonic-react/src/webchat/webview/index.tsx

```diff
@@ -16,11 +20,14 @@ export const WebviewContainer = () => {
   const { closeWebview, getThemeProperty, webchatState } =
     useContext(WebchatContext)
 
-  const webviewRequestContext = {
-    closeWebview: async (options?: CloseWebviewOptions) =>
-      await closeWebview(options),
+  const webviewRequestContext: WebviewRequestContextType = {
     params: webchatState.webviewParams || ({} as Record<string, any>),
```

Comment:
Should this type be updated also as `Record<string, string>`?

Reply:
done